mp/src/game/shared/sdk/sdk_gamerules.cpp

```diff
@@ -500,6 +500,9 @@ void CSDKGameRules::CalculateSlowMoForPlayer(CSDKPlayer* pPlayer)
 		return;
 	}
 
+	if (enginetrace->PointOutsideWorld(pPlayer->EyePosition()))
```

Comment:
This could false positive sometimes when players are stunting around, but I did my best to minimize the occurrence of that so I'm willing to tolerate a sometimes failure to set slowmo type in order to avoid that assert.

Reply:
Well, you can't do PVS outside the world anyway, which is what the assertion is all about
Previous behaviour was just to treat them as if they saw no one at that time, i.e. disable slowmo.
The new behaviour of just skipping slowmo calculation for the frame is better imo.